.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,54 @@
+name: Integration test
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+# Cancel the job if new commits are pushed
+# https://stackoverflow.com/questions/66335225/how-to-cancel-previous-runs-in-the-pr-when-you-push-new-commitsupdate-the-curre
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:  
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ["3.10", "3.12"]
+        package: ["mudata", "spatialdata", "scirpy", "muon", "scanpy", "squidpy", "scvi-tools"]
+
+    defaults:
+      run:
+        shell: bash -el {0}
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: scverse/${{ matrix.package }}
+  
+      - name: Install Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Install UV
+        uses: hynek/setup-cached-uv@v2
+
+      - name: Install AnnData + test dep
+        run: |
+          printf "numba>=0.60.0\nnumpy<2.0" | tee /tmp/constraints.txt
+          uv pip install --compile --system ".[test]" -c /tmp/constraints.txt -v 
+          uv pip install --compile --system git+https://github.com/scverse/anndata 
+
+      - name: Env list
+        run: uv pip freeze
+
+      - name: Run test
+        run: pytest
```

Comment:
Issue with tagging of anndata team to ensure our supervision

Reply:
I usually do things like that this way:
1. if there is an error
2. use GH API to search open issues containing some marker text (we control the issue text body, so we can e.g. add a HTML comment with a marker like `<!-- !!!!SCVERSE INTEGRATION TESTING MARKER!!!! -->`)
3. if no open one is found, open a new one, otherwise edit the existing one (in this case maybe no editing is necessary, as people can check out the newest failures here themselves)

Reply:
Ah, ok, I went with using the title